tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -436,13 +439,13 @@ def test_conv_features(
 )
 @pytest.mark.parametrize(
     "weights_dtype",
-    [ttnn.bfloat16, ttnn.bfloat8_b],
+    [ttnn.bfloat16],
 )
 @pytest.mark.parametrize(
     "activations_dtype",
-    [ttnn.bfloat16, ttnn.bfloat8_b],
+    [ttnn.bfloat16],
 )
-@pytest.mark.parametrize("auto_shard", [True], ids=["auto_shard"])
+@pytest.mark.parametrize("auto_shard", [False], ids=["no_auto_shard"])
```

Comment:
why did you change auto shard again there should be both options
same questino for data formats?

Reply:
I had removed it during debugging. Will add back.

Reply:
but it's bad even on main, should be two options